### DISCUSSION TITLE
Images not getting processed from archive

### DISCUSSION THREAD

**Discussion by WorldTeacher** [Difegue/LANraragi#428] [Q&A] [answered]
**LRR Version and OS**
Windows 10
Latest 07.7
**Bug Details**
importing archives results in missing images. Some Images get loaded, while others don't load at all, while I can view them just fine in the original archive. Trying to download the image to view it manually gives me an error from my image viewer "the image is corrupted" 
Any thoughts on how to correct/fix this?
**Matching Logs**
I didn't find any, might be missing some though

**Screenshots**
![img not loading](https://user-images.githubusercontent.com/41587052/111702435-8d999500-883c-11eb-81e6-a3b79ce69b9e.png)

**Comment by Difegue** [comment, marked as answer]
Found out from further support on Discord that this was seemingly caused by the archives being in .RAR format.
